src/dashboard/frontend/src/index.css

```
@@ -135,7 +220,7 @@
 @layer base {
   body {
     @apply bg-background text-foreground;
-    font-family: "DM Sans", system-ui, sans-serif;
+    font-family: var(--font-sans), system-ui, sans-serif;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)